README.md

```diff
@@ -1,12 +1,26 @@
 # Setup
+
+#### Install PyTorch
```

Comment:
I would suggest a link to add a link to 
https://pytorch.org/get-started/locally/ 
and to 
https://developer.nvidia.com/cuda-downloads 
suggesting to install a cuda toolkit supported by torch, or a torch matching the toolkit. Makes the cupy dependency easier.